tests/manage/z_cluster/test_must_gather.py

```diff
@@ -70,10 +77,39 @@ def finalizer():
 
         request.addfinalizer(finalizer)
 
+    @pytest.mark.polarion_id("OCS-1583")
     def test_must_gather(self):
         """
         Tests functionality of: oc adm must-gather
 
+        """
+        self.collect_must_gather()
+
+    @pytest.mark.polarion_id("OCS-2328")
+    def test_must_gather_worker_node_down(self, nodes):
+        """
+        Collect must-gather OCS logs when a worker node is down
+
+        """
+        self.reboot_node = True
+
+        logger.info('Get all worker nodes and choose random from worker nodes list')
+        worker_nodes = get_worker_nodes()
+        worker_node = worker_nodes[randint(0, len(worker_nodes) - 1)]
+
+        # Stop one worker node
+        nodes.stop_nodes(get_node_objs([worker_node]))
+
+        # Collect must gather and check content
+        self.collect_must_gather()
+
+        # Start worker node
+        nodes.start_nodes(get_node_objs([worker_node]))
+
+    def collect_must_gather(self):
```

Comment:
This name does not describe the function - i would got for something like `run_must_gether_test`, as it this main testing function

Reply:
`validate_must_gather_content` is ok?